client/X11/xf_event.c

```diff
@@ -366,6 +366,9 @@ static BOOL xf_event_MotionNotify(xfContext* xfc, XEvent* event, BOOL app)
 	if (xfc->use_xinput)
 		return TRUE;
 
+	if(xfc->floatbar && !(app))
```

Comment:
This stuff is part of XEvent handling, couldn't that be done in that function?

Reply:
I need to know in the floatbar where the mouse pointer is within the root window to decide wether or not the floatbar needs to be shown or hidden. That value is checked outside of any Event handling as the main loop calls the xf_floatbar_hide_and_show function regulary. 
Alternativeley i could store the last x, y postion of the pointer outside of my xfFloatbar struct, but i did not want to mess around to much with the existing structures.

Reply:
@mmattes Ah, ok, then that is a valid case.